unpackaged/config/trial/package.xml

```diff
@@ -36,6 +37,10 @@
     </types>
     <types>
         <members>%%%NAMESPACE%%%Grade_Enrollment__c.Grade_Number__c</members>
+        <members>%%%NAMESPACE%%%Grade_Enrollment__c.Education_History__c</members>
+        <members>%%%NAMESPACE%%%Education_History__c.End_Reason__c</members>
+        <members>%%%NAMESPACE%%%Education_History__c.School_Enrollment_Status__c</members>
+        <members>%%%NAMESPACE%%%Education_History__c.Start_Reason__c</members>
```

Comment:
@m-kinkade Since we are not adding these fields as part of the metadata in this package, we do not need to have these entries in the package XML. Were we to, please be mindful of the namespace tokens. Education History is part of EDA's namespace and not K-12 Architecture Kit's.
```suggestion
```